web/client/translations/data.vi-VN.json

```diff
@@ -144,6 +144,7 @@
             "noRecordsMatched": "Không có bản ghi phù hợp",
             "noResultsText": "Không có kết quả",
             "notAvailable": "Không có sẵn",
+            "invalidUrlHttpProtocol": "URL không khớp với giao thức HTTP của MapStore {protocol}",
```

Comment:
We usually translate only the default languages supported by mapstore DE, US, ES, FR and IT so this could be removed